src/main/kotlin/org/kamiblue/client/module/modules/chat/FancyChat.kt

```diff
@@ -14,14 +14,14 @@ internal object FancyChat : Module(
     showOnArray = false,
     modulePriority = 100
 ) {
-    private val uwu = setting("uwu", true)
-    private val leet = setting("1337", false)
-    private val mock = setting("mOcK", false)
-    private val green = setting(">", false)
-    private val blue = setting("`", false)
-    private val randomSetting = setting("Random Case", true, { mock.value })
-    private val commands = setting("Commands", false)
-    private val spammer = setting("Spammer", false)
+    private val uwu = setting("uwu", false, description = "Uwufy all of your messages")
+    private val leet = setting("1337", false, description = "Exchange certain characters for other similar looking characters")
```

Comment:
Neither of these are really useful. `uwufy` is only useful if the person already understands what `uwu` means in this context